.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: python
```

Comment:
I think this might cause the 3.7 build to break (because travis dosen't have a 3.7 tarball yet)

Reply:
yes, you need to let conda to handle python 3.7 the usual way, see https://github.com/astropy/astropy/pull/7607/files

Reply:
either way you're using ci-helpers to set up your environment, so not really using whatever python version of travis you're setting, no point having the language set to 'python' imo.